cmd/otelcol/Dockerfile

```diff
@@ -7,10 +9,16 @@ COPY otelcol /
 # copied with the execute bit lost (see #1317)
 RUN chmod 755 /otelcol
 
-FROM scratch
-COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+FROM quay.io/signalfx/signalfx-agent:${SMART_AGENT_VERSION_NUMBER}
```

Comment:
I would advise against adding this dependency.  Using the standalone agent bundle and running the patch-interpreter script should be fine (what the deb and rpm jobs do): https://github.com/signalfx/signalfx-agent#linux-standalone-targz

Reply:
https://github.com/signalfx/splunk-otel-collector/blob/a4dd11d6d68bfe3d408c49dd33b8cf260f0aa684/internal/buildscripts/packaging/fpm/common.sh#L68

Reply:
Ok, I'm now using alpine:3.12 as parent image.